### PR TITLE
Tweak CI configs a bit

### PR DESCRIPTION
**Pull request by Goooler**
## Description

A few tweaks.

## Checklist

- [x] PR description added
- [ ] tests are added
- [ ] KtLint has been applied on source code itself and violations are fixed
- [ ] [documentation](https://pinterest.github.io/ktlint/) is updated
- [ ] `CHANGELOG.md` is updated

In case of adding a new rule:
- [ ] Rule is added to [rules documentation](https://pinterest.github.io/ktlint/rules/standard/)

### CODE REVIEW

File: .github/workflows/gradle-pr-build.yml [hunk 1]
```diff
@@ -3,6 +3,11 @@ name: PR Build
 on:
   pull_request:
 
+env:
+  ORG_GRADLE_PROJECT_signingKey : ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEY }}
+  ORG_GRADLE_PROJECT_signingKeyId: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEYID }}
+  ORG_GRADLE_PROJECT_signingKeyPassword : ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGPASSWORD }}
```
Comment (Goooler) [inline .github/workflows/gradle-pr-build.yml:9]: @shashachu I believe these secrets are unnecessary for PR builds for now, right?

File: .github/workflows/gradle-wrapper-validation.yml [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+  push:
+    paths:
+      - 'gradlew'
+      - 'gradlew.bat'
+      - 'gradle/wrapper/'
+  pull_request:
+    paths:
+      - 'gradlew'
+      - 'gradlew.bat'
+      - 'gradle/wrapper/'
```
Comment (Goooler) [inline .github/workflows/gradle-wrapper-validation.yml:12]: If we can remove the check for `Validation`?

File: .github/workflows/gradle-pr-build.yml (hunk 1 above)
Comment (Goooler) [reply .github/workflows/gradle-pr-build.yml:9]: If so, I'll push a commit the remove them.

Comment (shashachu) [reply .github/workflows/gradle-pr-build.yml:9]: no, unfortunately, at least without some other tweaking. These being missing were the reason why the PR builds started failing once @paul-dingemans committed the version to be a non-SNAPSHOT version.

File: .github/workflows/gradle-wrapper-validation.yml (hunk 2 above)
Comment (shashachu) [reply .github/workflows/gradle-wrapper-validation.yml:12]: This is probably correct, but the validation is also very quick so I don't think it's a huge savings either.

Comment (Goooler) [reply .github/workflows/gradle-wrapper-validation.yml:12]: Cause there might be some flaky failures from this action.

File: .github/workflows/gradle-pr-build.yml (hunk 1 above)
Comment (Goooler) [reply .github/workflows/gradle-pr-build.yml:9]: Gotcha.

**Review by shashachu — APPROVED** [review]

**Comment by shashachu** [comment]
Ah, actually we might be stuck a little bit because I'm not sure if we can conditionally require status checks. So this can't be merged (except by me) because it's waiting for the Gradle Wrapper Validation to pass.

**Comment by shashachu** [comment]
@Goooler I wonder if instead we can just return success from the Validation step if the wrapper files haven't changed?

**Comment by Goooler** [comment]
Yep, that's reasonable, but I'm not sure how to implement it.

**Comment by paul-dingemans** [comment]
I am not sure whether this PR should be merged, but I can not do so at this moment at there still a Validation step which is not completed.

**Pull request merged**